e2e-tests/commands/insurance/check-policy-summary-list.js

```diff
@@ -111,21 +116,21 @@ const checkPolicySummaryList = ({
     },
     [TOTAL_SALES_TO_BUYER]: () => {
       const fieldId = TOTAL_SALES_TO_BUYER;
-      const expectedKey = FIELDS.CONTRACT_POLICY.MULTIPLE[fieldId].SUMMARY.TITLE;
+      const expectedKey = FIELDS.EXPORT_VALUE.MULTIPLE[fieldId].SUMMARY.TITLE;
 
       const expectedValue = formatCurrency(application.POLICY[fieldId]);
 
-      const expectedChangeLinkText = FIELDS.CONTRACT_POLICY.MULTIPLE[fieldId].SUMMARY.TITLE;
+      const expectedChangeLinkText = FIELDS.EXPORT_VALUE.MULTIPLE[fieldId].SUMMARY.TITLE;
```

Comment:
could we maybe dereference FIELDS.EXPORT_VALUE?

Reply:
Good shout, have updated so it's using `CONTRACT_POLICY` and `EXPORT_VALUE` without `FIELDS`. I don't think it go further than this into `SINGLE` and `MULTIPLE` as it will have less context in this file